File: POO/TP2/Practica Python/EJ9/Persona.py
class Persona:
    __edad: int
    __sexo: str
    __estudia: bool
    __trabaja: bool
    
    def __init__(self, edad, sexo, estudia, trabaja):
        self.__edad= edad
        self.__sexo= sexo
        self.__estudia= estudia
        self.__trabaja= trabaja
    
    def get_edad(self):
        return self.__edad
    
    def set_edad(self,edad):
        self.__edad=edad
        
    def get_sexo(self):
        return self.__sexo
    
    def set_sexo(self,sexo):
        self.__sexo=sexo
        
    def get_estudia(self):
        return self.__estudia
    def set_estudia(self,estudia):
        self.__estudia=estudia
    
    def get_trabaja(self):
        return self.__trabaja
    def set_trabaja(self,trabaja):
        self.__trabaja=trabaja

File: POO/TP2/Practica Java/EJ8/Persona.java
package EJ8;

public class Persona {
    private Integer edad;
    private String sexo;
    private boolean estudia;
    private boolean trabaja;

    public Persona(Integer edad, String sexo, boolean estudia, boolean trabaja) {
        this.edad = edad;
        this.sexo = sexo;
        this.estudia = estudia;
        this.trabaja = trabaja;
    }

    public Integer getEdad() {
        return this.edad;
    }

    public void setEdad(Integer edad) {
        this.edad = edad;
    }

    public String getSexo() {
        return this.sexo;
    }

    public void setSexo(String sexo) {
        this.sexo = sexo;
    }

    public boolean getEstudia() {
        return this.estudia;
    }

    public void setEstudia(boolean estudia) {
        this.estudia = estudia;
    }

    public boolean getTrabaja() {
        return this.trabaja;
    }

    public void setTrabaja(boolean trabaja) {
        this.trabaja = trabaja;
    }

}
